products/jbrowse-web/src/SessionLoader.ts

```diff
@@ -80,10 +80,6 @@ const SessionLoader = types
     ),
   })
   .volatile(() => ({
-    /**
-     * #volatile
-     */
-    pluginsLoaded: false,
```

Comment:
can you explain the removal of the pluginsLoaded flag? are there any options to keep it?

Reply:
if we can truly derive the ready-ness of this flag from conditions in the data, it could be removed but there is some somewhat complex logic that I'll just add some other comments for

Reply:
I've restored it as a view that encapsulates both runtime and session plugins being ready